src/main.cpp

```diff
@@ -984,15 +984,15 @@ int64_t GetProofOfWorkReward(int64_t nFees)
 }
 //const int DAILY_BLOCKCOUNT =  1440;
 // miner's coin stake reward based on coin age spent (coin-days)
-int64_t GetProofOfStakeReward(int64_t nCoinAge, int64_t nFees)
+int64_t GetProofOfStakeReward(int64_t nCoinAge, int64_t nFees) //-----This is part of the staking code to be changed-----
 {
     int64_t nRewardCoinYear;
 
     nRewardCoinYear = MAX_MINT_PROOF_OF_STAKE;
 
     int64_t nSubsidy;
     
-    if(pindexBest->nHeight > LAST_OLD_POS_BLOCK)
+    if(nCoinAge > 24)         // The condition will be changed so that reward conditions do not rely on network weight
```

Comment:
I'm not sure this is the right place for this change. The line here used to be a check point to cater for the pre-fork vs post-fork situation (for which I don't know anything about).
Plus, were you able to find tests for the current staking algorithm that we could run to ensure that the change you're proposing holds up?

Reply:
I do not have any current way to test any changes made. That is part of the problem why this is taking longer than expected. Most of the comments where for my own sake to keep track of what is going on. If there is a way to delete this pull request and move it to an open discussion that would be great. I would like to form a team on this issue to help the process go more smoothly.